src/main/java/com/petproject/pomodoro/dto/UserDto.java

```diff
@@ -0,0 +1,59 @@
+package com.petproject.pomodoro.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+/**
+ * Created by Student on 15.06.2020
+ */
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDto {
+    private String email;
```

Comment:
Don't know for sure, but maybe we would need to know userId as well -> to show in URL to our endpoints (users/1/pomodoros, for example)